ONSdigital/sdx-store
# Multiple queues

**Changes**
Handles connecting to a second queue, should the first fail

**How to test**
Run as part of sdx-compose, submit survey, should work as normal
Edit port of first rabbit url settings in sdx-store to incorrect value, submit survey, should fallback to the second url

**Who can test**
Anyone but @elliott-jenkins

**Review**
Code structure
How to automate fallover test
